fabric/build.gradle

```diff
@@ -0,0 +1,69 @@
+plugins {
+	id "com.github.johnrengelman.shadow" version "7.1.2"
+}
+
+architectury {
+	platformSetupLoomIde()
+	fabric()
+}
+
+loom {
+	accessWidenerPath = project(":common").loom.accessWidenerPath
```

Comment:
is this useless? conditional-mixin does not and will not need any access wideners

Reply:
yes, seems like I missed that bit